tests/mock/Concrete/FakeOrder.php

```diff
@@ -0,0 +1,515 @@
+<?php
+
+namespace Mundipagg\Core\Test\Mock\Concrete;
+
+use Mundipagg\Core\Kernel\Interfaces\PlatformInvoiceInterface;
+use Mundipagg\Core\Kernel\ValueObjects\Id\OrderId;
+use Mundipagg\Core\Kernel\ValueObjects\OrderState;
+use Mundipagg\Core\Kernel\ValueObjects\OrderStatus;
+use Mundipagg\Core\Payment\Aggregates\Customer;
+use Mundipagg\Core\Payment\Aggregates\Item;
+use Mundipagg\Core\Payment\Aggregates\Payments\AbstractPayment;
+use Mundipagg\Core\Payment\Aggregates\Shipping;
+
+class FakeOrder
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The class FakeOrder has 45 public methods and attributes. Consider reducing the number of public items to less than 45.](https://app.codacy.com/manual/mundipagg/ecommerce-module-core/pullRequest?prid=4926694)